src/backend/distributed/shared_library_init.c

```diff
@@ -346,7 +346,9 @@ _PG_init(void)
 	{
 		DoInitialCleanup();
 	}
-	columnar_init();
+
+
+	load_file("columnar.so", false);
```

Comment:
could use a comment, e.g.
```c
/*
 * For convenience and backwards compatibility, we avoid users having to add both
 * citus and columnar to shared_preload_libraries by loading columnar.so as part of
 * loading citus.so.
 */
```

maybe also some compile-time variable for the file name.

Reply:
add comment there as suggested